client/src/components/DescriptiveTextButton/DescriptiveTextButton.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./DescriptiveTextButton.css";
+import { PropTypes } from "prop-types";
+
+export const DescriptiveTextButton = ({ desc, data, onClick }) => {
+  return (
+    <div className="no-account-text">
+      {desc}
+      <button className="no-account-button" onClick={onClick}>
+        {data}
```

Comment:
We can probably use a more descriptive variable here rather than just `data`.

Reply:
Fixed: Changed to `data` to `nextPageText`